packages/tools/src/tools/CrosshairsTool.ts

```diff
@@ -151,6 +158,9 @@ export default class CrosshairsTool extends AnnotationTool {
     this._getReferenceLineSlabThicknessControlsOn =
       toolProps.configuration?.getReferenceLineSlabThicknessControlsOn ||
       defaultReferenceLineSlabThicknessControlsOn;
+
+    this._applySlabThicknessToOnlyFirstActor =
```

Comment:
Is there any case where we want like 2/3 actors mipped? I'm asking because then we should make the config item an array, and use that as a filter for what to MIP.

The tool configuration is public API, so i'd rather it not change again next time we realised we need this?